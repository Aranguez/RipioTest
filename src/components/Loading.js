import React from 'react';

const Loading = () => (
    <div>
        <h4 className="t-azul text-center">
            <i className="fa fa-spinner fa-spin"></i> Enviando Bitcoins
        </h4>
    </div>
)

export default Loading;
