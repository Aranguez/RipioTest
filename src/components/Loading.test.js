import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

describe('loading', () => {
    const loading = shallow(<Loading/>)
    
    it('renders properly ', () => {
        expect(loading).toMatchSnapshot()
    });
})