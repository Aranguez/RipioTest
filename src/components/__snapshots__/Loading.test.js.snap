// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading renders properly  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h4
        className="t-azul text-center"
      >
        <i
          className="fa fa-spinner fa-spin"
        />
         Enviando Bitcoins
      </h4>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <i
            className="fa fa-spinner fa-spin"
          />,
          " Enviando Bitcoins",
        ],
        "className": "t-azul text-center",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "fa fa-spinner fa-spin",
          },
          "ref": null,
          "rendered": null,
          "type": "i",
        },
        " Enviando Bitcoins",
      ],
      "type": "h4",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h4
          className="t-azul text-center"
        >
          <i
            className="fa fa-spinner fa-spin"
          />
           Enviando Bitcoins
        </h4>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <i
              className="fa fa-spinner fa-spin"
            />,
            " Enviando Bitcoins",
          ],
          "className": "t-azul text-center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-spinner fa-spin",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          " Enviando Bitcoins",
        ],
        "type": "h4",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
